packages/core/EventManager.ts

```diff
@@ -554,6 +555,7 @@ export default class EventManager {
         [] as DestinationCalendar[]
       );
       for (const destination of destinationCalendars) {
+        if (eventCreated) break;
```

Comment:
If a calendar event was created then set `eventCreated` to `true`. If the event was created on the first destination calendar then break out of the loop.